packages/firestore/src/api/database.ts

```diff
@@ -285,6 +286,43 @@ export function configureFirestore(firestore: Firestore): void {
   );
 }
 
+/**
+ * Attempts to enable LRU garbage collection for memory persistence.
+ *
+ * Must be called before any other functions (other than
+ * {@link initializeFirestore}, {@link (getFirestore:1)} or
+ * {@link clearIndexedDbPersistence}.
+ *
+ * By default, any documents that are not part of an active query result or
+ * with no mutation attached to them are removed from memory immediately.
```

Comment:
Suggest "... result or
that have no mutation attached to them are removed from memory immediately.